trackintel/preprocessing/triplegs.py

```diff
@@ -1,6 +1,20 @@
+from shapely.geometry import LineString
+from simplification.cutil import simplify_coords#, simplify_coordsvw
```

Comment:
I think this breaks the CI, as simplification is not added to setup.py / requirements.txt.